lib/Conversion/LLHDToLLVM/LLHDToLLVM.cpp

```diff
@@ -2418,6 +2418,23 @@ struct CombConcatOpConversion : public ConvertToLLVMPattern {
 };
 } // namespace
 
+namespace {
+/// Lower a comb::ReplicateOp operation to the LLVM dialect.
+struct CombReplicateOpConversion
+    : public ConvertOpToLLVMPattern<comb::ReplicateOp> {
+  using ConvertOpToLLVMPattern<comb::ReplicateOp>::ConvertOpToLLVMPattern;
+
+  LogicalResult
+  matchAndRewrite(comb::ReplicateOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    std::vector<Value> inputs(op.getMultiple(), op.getInput());
+    rewriter.replaceOpWithNewOp<comb::ConcatOp>(op, inputs);
+    return success();
```

Comment:
Nice approach lowering to `ConcatOp` so the conversion for that op can then pick up.

Reply:
Yeah, as long as we don't have a specialized pattern to get higher performance, we can just reuse the ConcatOp pattern and directly benefit from any improvements done there.